operator/v1/types_ingress.go

```diff
@@ -376,6 +386,21 @@ type LoadBalancerStrategy struct {
 	// +required
 	Scope LoadBalancerScope `json:"scope"`
 
+	// allowedSourceRanges specifies an allowlist of IP address ranges to which
+	// access to the load balancer should be restricted.  Each range must be
+	// specified using CIDR notation (e.g. "10.0.0.0/8" or "fd00::/8"). If no range is
+	// specified, "0.0.0.0/0" for IPv4 and "::/0" for IPv6 are used by default,
+	// which allows all source addresses.
+	//
+	// To facilitate migration from earlier versions of OpenShift that did not have the
+	// allowedSourceRanges field, the service.beta.kubernetes.io/load-balancer-source-ranges
+	// annotation on the load balancer service takes precedence on OpenShift v4.12
```

Comment:
which service is the load balancer service?

Reply:
To David's point, we can clarify as follows:
```suggestion
 	// To facilitate migration from earlier versions of OpenShift that did
	// not have the allowedSourceRanges field, you may set the
	// service.beta.kubernetes.io/load-balancer-source-ranges annotation on
	// the "router-<ingresscontroller name>" service in the
	// "openshift-ingress" namespace, and this annotation will take
	// precedence over allowedSourceRanges on OpenShift 4.12.
```

Reply:
I applied Miciah's suggestion.